src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -87,9 +87,9 @@ if [ ! -z ${RunCrossGen2+x} ]%3B then
         local heapVerifyModeToRestore=$COMPlus_HeapVerify;
         local readyToRunModeToRestore=$COMPlus_ReadyToRun;
         local gcstandaloneModeToRestore=$COMPlus_GCName;
-        export COMPlus_GCStress=
-        export COMPlus_HeapVerify=
-        export COMPlus_ReadyToRun=
+        unset COMPlus_GCStress
```

Comment:
I am wondering how this could make any difference. Exporting `COMPlus_GCStress=''` should still disable GC stress, shouldn't it?

Reply:
Well, according to JanV setting the environment variable in bash to an empty string doesn't have the expected effect of removing the variable. It probably matters not as you've already refactored the entire logic for manipulating these variables.